go/service/chat_local.go

```diff
@@ -328,6 +329,9 @@ func (h *chatLocalHandler) GetThreadNonblock(ctx context.Context, arg chat1.GetT
 			SessionID: arg.SessionID,
 			Thread:    remoteThread,
 		})
+
+		// This means we transmitted with success, so cancel local thread
+		cancel()
```

Comment:
Used to do `defer cancel()` at the top, but I didn't want it to do that if we hit an error.